examples/vae.py

```diff
@@ -17,6 +19,11 @@
 from scipy.misc import imsave
 from tensorflow.examples.tutorials.mnist import input_data
 
+try:
```

Comment:
instead of try/except, I find [this approach](https://github.com/blei-lab/edward/blob/master/examples/tf_convolutional_vae.py#L101-L107) a bit more readable, and also letting `IMG_DIR` be a variable.